ui/components/component-library/text-field-search/text-field-search.js

```diff
@@ -100,10 +100,6 @@ TextFieldSearch.propTypes = {
    * Attributes applied to the `input` element.
    */
   inputProps: PropTypes.object,
-  /**
-   * FormTextField accepts all the props from TextField and Box
-   */
-  ...TextField.propTypes,
```

Comment:
Now that we are using the deprecated JS version here I think we can revert these changes to the propTypes here. So the only changes to this file will be the import path of the TextField e.g.
```
import { TextField, TEXT_FIELD_TYPES } from '../text-field/deprecated';
```